js/okex.js

```diff
@@ -3544,6 +3546,47 @@ module.exports = class okex extends Exchange {
         };
     }
 
+    async fetchBorrowRates (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetAccountInterestRate (params);
+        const timestamp = this.milliseconds ();
+        const data = this.safeValue (response, 'data');
+        const rates = [];
+        for (let i = 0; i < data.length; i++) {
+            const rate = data[i];
+            const currency = this.safeCurrencyCode (this.safeString (rate, 'ccy'));
+            rates[currency] = {
+                'currency': currency,
+                'rate': this.safeNumber (rate, 'interestRate'),
+                'period': 86400000,
+                'timestamp': timestamp,
+                'datetime': this.iso8601 (timestamp),
+                'info': rate,
+            };
+        }
+        return rates;
+    }
+
+    async fetchBorrowRate (code, params = {}) {
+        await this.loadMarkets ();
+        const currencyId = this.currency (code)['id'];
```

Comment:
Please, split this into several lines:

```JavaScript
const currency = this.currency (code);
const request = {
    'ccy': currency['id'],
}
```

We will have to unify parseBorrowRate eventually, and `currency` will go as the second argument.
Our regular naming in most of parsers:

```JavaScript
code = currency['code']
currencyId = currency['id']
```